### DISCUSSION TITLE
Sugestão (Map-OS v5)

### DISCUSSION THREAD

**Discussion by MilsonElias** [RamonSilva20/mapos#1615] [General]
Olá, faz um bom tempo que estou trabalhando em melhorias e mudanças no Map-OS, e gostaria de compartilhar com vocês.
Vou postar aqui minhas ideias e mudanças no sistema. Espere que os criadores aceitem minhas ideias, e quem sabe eu possa colaborar com a equipe.

Novo Logo:
![logo map](https://user-images.githubusercontent.com/92489803/150892721-7d8d8e8a-d126-4bbc-8679-8ff1ab85ebfb.png)

Login:
![Painel imagem 9](https://user-images.githubusercontent.com/92489803/150894687-3121b057-36e6-47d6-806c-9b0557356196.jpg)

Painel Principal:
![Painel imagem 1](https://user-images.githubusercontent.com/92489803/150892553-6132a675-87d7-495e-8777-659167a7aec1.jpg)
![Painel imagem 2](https://user-images.githubusercontent.com/92489803/150892480-afe086a1-cade-43ec-96cd-2a851a1a46b9.jpg)
![Painel imagem 3](https://user-images.githubusercontent.com/92489803/150892486-5c4fc74c-799c-4a22-a5da-b1efb0e7c73e.jpg)

Modo Phone:
![Painel imagem 4](https://user-images.githubusercontent.com/92489803/150892488-20467274-c5ef-4f0a-8ffe-2d17a070a5ce.jpg)
![Painel imagem 8](https://user-images.githubusercontent.com/92489803/150892495-1c9e89c3-adaf-4146-83af-3e06be7f32c7.jpg)
![Painel imagem 6](https://user-images.githubusercontent.com/92489803/150892491-09dc1fd8-9ed5-434d-87d2-6d09bf129196.jpg)

Tema Escuro:
![Painel Black 1](https://user-images.githubusercontent.com/92489803/150895438-87cb6b05-84c7-4b9c-b6de-4e9180990b91.jpg)
![Painel Black 2](https://user-images.githubusercontent.com/92489803/150895441-a3294740-a239-4140-9cf0-fcf0409d8ac7.jpg)
![Painel Black 3](https://user-images.githubusercontent.com/92489803/150895442-96493c44-ee3a-4ec1-ab44-8bf3f426971e.jpg)
![Painel Black 4](https://user-images.githubusercontent.com/92489803/150895444-83c2733b-2e46-48f0-9a50-f20d845c39ea.jpg)
![Painel Black 5](https://user-images.githubusercontent.com/92489803/150895445-5b0765f9-dc1b-41d8-bfc3-04dd0507905c.jpg)

**Comment by radsonpatrick** [comment]
Sem palavras para descrever o quanto isso ficou bom.. deu uma vida nova a interface do programa....

**Comment by Pr3d4dor** [comment]
@MilsonElias Parabéns pelo novo design, ficou muito bom.

O mapos é um projeto de código aberto, então suas melhorias seriam muito mais do que bem-vindas.

Basta abrir uma PR e a equipe irá analisar e se estiver tudo OK, será feito o merge e lançado nova versão.

**Reply by MilsonElias** [reply]
Ok, obrigado pela dica, estou revisando tudo pra fazer o PR em breve.

**Reply by duuhmenezes** [reply]
shoooooooooooooooooooooooooooooooooow demais !!!

**Comment by duuhmenezes** [comment]
Parabéns, ficou foda demais, parabéns a todos os envolvidos !

**Comment by brtechst** [comment]
Mano ta lindo demais!!!!

Por favor, manda logo essa PR, porque essa interface ta sinistra!!!!

fez mudanças na aba de OS? Se sim, poderia mandar print e como está?

**Comment by SuperPorte** [comment]
Ficou fantástico o visual.
Se der pra por 2 botões de whatsapp (um para WEB e outro para API) seria mais show ainda.
Incluir cadastros DENTRO da OS também seria excelente!!!

**Comment by fabianocruzdf** [comment]
@MilsonElias cara, excelente! O mapos precisa muito dessa "cara" nova.
Torcendo aqui pra ser liberado o quanto antes!

**Comment by SuperPorte** [comment]
Então.....

_ _ _
Leonardo B. Bernardi
Bernardi Computadores
(27) 992-882-865
***@***.***

De: BESystem Serviços de Informática
Enviado:quinta-feira, 3 de fevereiro de 2022 14:01
Para: RamonSilva20/mapos
Cc:bernarcom; Comment
Assunto: Re: [RamonSilva20/mapos] Sugestão (Map-OS v5) (Discussion #1615)

Creio que isso não seja mais necessário amigo. Na api (pelo navegador do PC), quando vc clica em iniciar conversa, se não tiver o Whatsapp ele dá a opção de fazer o Download, ou logo abaixo para usar o Whatsapp web, que abre a versão web no navegador. Faz um teste em um PC que não tenha Whatsapp para ver.
Lembrando, tem que clicar no Iniciar conversa da primeira tela, mesmo sem ter o aplicativo instalado.

—
Reply to this email directly, view it on GitHub, or unsubscribe.
Triage notifications on the go with GitHub Mobile for iOS or Android.

**Comment by SuperPorte** [comment]
Desculpa o email vazio, apertei CTRL+ENTER

Então...
É que quando se faz várias repostas aos clientes, a versão web recarrega tuuuudo de novo a cada nova mensagem, e na API esse processo fica mais rápido

_ _ _
Leonardo B. Bernardi
Bernardi Computadores
(27) 992-882-865
***@***.***

De: BESystem Serviços de Informática
Enviado:quinta-feira, 3 de fevereiro de 2022 14:01
Para: RamonSilva20/mapos
Cc:bernarcom; Comment
Assunto: Re: [RamonSilva20/mapos] Sugestão (Map-OS v5) (Discussion #1615)

Creio que isso não seja mais necessário amigo. Na api (pelo navegador do PC), quando vc clica em iniciar conversa, se não tiver o Whatsapp ele dá a opção de fazer o Download, ou logo abaixo para usar o Whatsapp web, que abre a versão web no navegador. Faz um teste em um PC que não tenha Whatsapp para ver.
Lembrando, tem que clicar no Iniciar conversa da primeira tela, mesmo sem ter o aplicativo instalado.

—
Reply to this email directly, view it on GitHub, or unsubscribe.
Triage notifications on the go with GitHub Mobile for iOS or Android.

**Comment by willph** [comment]
> Olá, faz um bom tempo que estou trabalhando em melhorias e mudanças no Map-OS, e gostaria de compartilhar com vocês. Vou postar aqui minhas ideias e mudanças no sistema. Espere que os criadores aceitem minhas ideias, e quem sabe eu possa colaborar com a equipe.
> 
> Novo Logo: ![logo map](https://user-images.githubusercontent.com/92489803/150892721-7d8d8e8a-d126-4bbc-8679-8ff1ab85ebfb.png)
> 
> Login: ![Painel imagem 9](https://user-images.githubusercontent.com/92489803/150894687-3121b057-36e6-47d6-806c-9b0557356196.jpg)
> 
> Painel Principal: ![Painel imagem 1](https://user-images.githubusercontent.com/92489803/150892553-6132a675-87d7-495e-8777-659167a7aec1.jpg) ![Painel imagem 2](https://user-images.githubusercontent.com/92489803/150892480-afe086a1-cade-43ec-96cd-2a851a1a46b9.jpg) ![Painel imagem 3](https://user-images.githubusercontent.com/92489803/150892486-5c4fc74c-799c-4a22-a5da-b1efb0e7c73e.jpg)
> 
> Modo Phone: ![Painel imagem 4](https://user-images.githubusercontent.com/92489803/150892488-20467274-c5ef-4f0a-8ffe-2d17a070a5ce.jpg) ![Painel imagem 8](https://user-images.githubusercontent.com/92489803/150892495-1c9e89c3-adaf-4146-83af-3e06be7f32c7.jpg) ![Painel imagem 6](https://user-images.githubusercontent.com/92489803/150892491-09dc1fd8-9ed5-434d-87d2-6d09bf129196.jpg)
> 
> Tema Escuro: ![Painel Black 1](https://user-images.githubusercontent.com/92489803/150895438-87cb6b05-84c7-4b9c-b6de-4e9180990b91.jpg) ![Painel Black 2](https://user-images.githubusercontent.com/92489803/150895441-a3294740-a239-4140-9cf0-fcf0409d8ac7.jpg) ![Painel Black 3](https://user-images.githubusercontent.com/92489803/150895442-96493c44-ee3a-4ec1-ab44-8bf3f426971e.jpg) ![Painel Black 4](https://user-images.githubusercontent.com/92489803/150895444-83c2733b-2e46-48f0-9a50-f20d845c39ea.jpg) ![Painel Black 5](https://user-images.githubusercontent.com/92489803/150895445-5b0765f9-dc1b-41d8-bfc3-04dd0507905c.jpg)

Previsão de quando enviara a PR?

O grupo no Whatsapp é https://chat.whatsapp.com/GkhxIyWVMqT3bHaSiIgO6e caso queira.

**Comment by SuperPorte** [comment]
Boa tarde galera!!
Outra ideia para a versão 5....
impressão de comprovantes de lançamento no financeiro.
Tipo, pagamento de condomínio, dá pra lançar, mas não tem como imprimir um "recibo" ou algo do gênero...
